### ISSUE TITLE
The oauth2login samples should show OIDC logout configuration

### ISSUE THREAD

**Issue by jzheaux** [spring-projects/spring-security#7292]
Related to https://github.com/spring-projects/spring-security/issues/7285#issuecomment-523240651

**Comment by ThomasVitale** [comment]
@jzheaux can I help with this task?

**Comment by jzheaux** [comment]
Yes, @ThomasVitale, that would be most welcome. We're moving our samples over to https://github.com/spring-projects/spring-security-samples, so I'm going to transfer this issue over there.

I think it would be helpful to have a new sample that's focused on OIDC, including logout. The existing oauth2Login samples are a showcase of common providers, but not all of them support OIDC. Publishing the `OidcClientInitiatedLogoutSuccessHandler` in those samples could be confusing.

Would you be able to create a new sample?
